sw/device/lib/testing/mock_csr.cc

```diff
@@ -0,0 +1,25 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/mock_csr.h"
+
+#include "sw/device/lib/base/csr.h"
+
+extern "C" {
+uint32_t mock_csr_read(uint32_t addr) {
```

Comment:
nit: just for the sake of consistency with `EXPECT_CSR_*` macros (`addr` -> `csr`, or the other way around).

Reply:
Done (`addr` -> `csr`).